CHANGELOG.md

```diff
@@ -324,33 +306,33 @@ the APIs in these libraries are stable, and are ready for production use.
 <summary> Expand to see the full list of new GA libraries...</summary>
 <br>
 
-* [Dataplex](/google/cloud/dataplex/README.md)
-* [DialogFlow CX](/google/cloud/dialogflow_cx/README.md)
-* [DialogFlow ES](/google/cloud/dialogflow_es/README.md)
+- [Dataplex](/google/cloud/dataplex/README.md)
+- [DialogFlow CX](/google/cloud/dialogflow_cx/README.md)
+- [DialogFlow ES](/google/cloud/dialogflow_es/README.md)
 
 </details>
 
 ### [Bigtable](/google/cloud/bigtable/README.md)
 
-* feat(bigtable): add `AsyncWaitForConsistency()` helper for Table Admin ([#9310](https://github.com/googleapis/google-cloud-cpp/pull/9310))
-* doc: broken link in Bigtable README ([#9265](https://github.com/googleapis/google-cloud-cpp/pull/9265))
+- feat(bigtable): add `AsyncWaitForConsistency()` helper for Table Admin ([#9310](https://github.com/googleapis/google-cloud-cpp/pull/9310))
```

Comment:
These are automatically generated as part of the release process.  Should we fix the script too?

Reply:
In the bigger picture these fall into the same category (for me) as the generator producing code which is later fixed up by clang-format, and we shouldn't worry about it too much.  In the smaller picture, though, these are easier to fix, and I think we should do so as soon as someone notices and cares.